lib/main_common.pm

```diff
@@ -1717,6 +1751,21 @@ sub load_ssh_key_import_tests {
     loadtest "x11/ssh_key_verify";
 }
 
+sub load_common_opensuse_sle_tests {
+    load_autoyast_clone_tests           if get_var('CLONE_SYSTEM');
```

Comment:
I think you changing model here in dangerous way - before we had elsif chain , so if you get into one branch you don't have chance to get into another , with your approach we will be capable to  execute wicked and systemd testsuite in single job which not always make sense

Reply:
other way would be just put as prefix `return` :
`load_toolchain_tests                if get_var('TCM') || check_var('ADDONS', 'tcm');`